infrastructure/prod-deploy/deploy-app.sh

```diff
@@ -40,4 +40,4 @@ python ./infrastructure/replace_envs.py ./infrastructure/prod-deploy/app.yaml
 # that the apps deploy to the correct services.
 # See: https://cloud.google.com/sdk/gcloud/reference/app/deploy
 gcloud app deploy ./app.yaml --quiet --project=codewof
-gcloud app deploy ./cron.yaml --quiet --project=codewof
+gcloud app deploy ./infrastructure/prod-deploy/cron.yaml --quiet --project=codewof-dev
```

Comment:
Is this supposed to be `codewof-dev` or just `codewof`?

Reply:
oh yeah codewof. missed copypaste mistake thanks